pkg/metadata/kubernetes/kubernetes.go

```diff
@@ -53,7 +53,10 @@ func GetPayload() (metadata.Payload, error) {
 	if ok && apiErr.Code == http.StatusNotFound {
 		dr = &appsv1beta1.DeploymentList{}
 	} else if err != nil {
-		return nil, fmt.Errorf("failed to get deployments: %s", err)
+		// Allow Deployments API to fail, it's not available in all version and we
+		// can also parse this data from the replica-set name on the backend.
+		log.Warnf("Failed to retrieve Kubernetes deployments: %s", err)
```

Comment:
didn't notice but `log` package was not imported

Reply:
Doh!